src/helperFunctions.js

```diff
@@ -113,7 +113,7 @@ class helpers {
    * @returns {Promise.<error>}           string: error message, if any.
    */
   _updateCurrentUser () {
-    let self = this
+    const self = this
```

Comment:
I know that you didn't touch it, but `self` should not be necessary here, as `this` is automatically bound as context in lambda functions.

Reply:
Can we agree to change this in a follow up PR?

Reply:
👍